packages/veritone-widgets/src/widgets/ContentTemplate/index.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { get } from 'lodash';
+import { objectOf, any, func } from 'prop-types';
+import { ContentTemplate } from 'veritone-react-common';
+
+import widget from '../../shared/widget';
+
+class ContentTemplateWidget extends React.Component {
+  static propTypes = {
+    templateData: objectOf(any).isRequired,
```

Comment:
this should be consistent with the propTypes you defined in the component, i.e -

``` 
templateData: objectOf(
      shape({
        id: string,
        name: string.isRequired,
        status: string,
        definition: objectOf(any)
      })
    ).isRequired
```

Same applies for `initialTemplates`